reaper/cogs/auto_response.py

```diff
@@ -107,6 +107,9 @@ async def on_message(self, message):
             datetime.datetime.now(datetime.timezone.utc) - self.last_time
         ).total_seconds()
 
+        if not message.guild:
```

Comment:
What do these checks do? ^^"

Reply:
If the bot gets a DM, this code will run and fail. It's a harmless error message but this prevents it.

Reply:
Aight, thanks :D

Added separately 9808c8f9cbbff62235eca9c58a33cf7428e219f8 and a52d401796e81b64b9b64e83422795930bf0a87b